department-of-veterans-affairs/caseflow
# Ryan/appeals 17514/mst pact checkboxes

Resolves #{17514}

### Description
Added MST/PACT status checkboxes when starting intake process, during add issues.

### Acceptance Criteria
- [ X] Code compiles correctly

### Testing Plan
1. Start with user BVADWISE
2. Start intake process
3. Fill in the form and search for veteran 
4. Press add issues
5. Click any radio option
6. Observe that a MST and PACT checkbox appears.
7. Toggle checkboxes to ensure they work.


### User Facing Changes
 
<img width="642" alt="image" src="https://user-images.githubusercontent.com/110805785/233742216-7d9a3535-12c1-4c85-bd5d-09a80cb1e0d6.png">

<img width="636" alt="image" src="https://user-images.githubusercontent.com/110805785/233742336-9f63ca7a-b15e-4b21-ab39-3339ec88ef5e.png">
